test/lib/lbt/analyzer/SmartTemplateAnalyzer.js

```diff
@@ -2,7 +2,7 @@ const test = require("ava");
 const SmartTemplateAnalyzer = require("../../../../lib/lbt/analyzer/SmartTemplateAnalyzer");
 const ModuleInfo = require("../../../../lib/lbt/resources/ModuleInfo");
 const sinon = require("sinon");
-const esprima = require("esprima");
+const parseJS = require("../../../../lib/lbt/utils/parseUtils");
```

Comment:
`parseUtils`

Reply:
Thanks, done in https://github.com/SAP/ui5-builder/pull/619